test/testcamera.c

```diff
@@ -278,6 +308,8 @@ int SDL_AppIterate(void *appstate)
          */
          frame_current = frame_next;
          texture_updated = SDL_FALSE;
+    } else {
+        return SDL_APP_CONTINUE;
```

Comment:
This else branch should not be here.
When there's no next frame, you still want to render the current frame.

Because of this early return, the app is broken (tested on Windows).

Reply:
See  #10540